content/en/logs/processing/parsing.md

```diff
@@ -105,7 +105,7 @@ Here is the list of all the matchers and filters natively implemented by Datadog
 | `rubyhash`                                                     | Parses properly formatted Ruby hash (e.g. `{name => "John", "job" => {"company" => "Big Company", "title" => "CTO"}}`).                                    |
 | `useragent([decodeuricomponent:true/false])`                   | Parses a user-agent and returns a JSON object that contains the device, OS, and the browser represented by the Agent. [Check the User Agent processor][1]. |
 | `querystring`                                                  | Extracts all the key-value pairs in a matching URL query string (e.g. `?productId=superproduct&promotionCode=superpromo`).                                |
-| `decodeuricomponent`                                           | This core filter decodes URI components.                                                                                                                  |
+| `decodeuricomponent`                                           | This core filter decodes URI components. It can be useful to decode UTF-8 characters (e.g. `%2Fservice%2Ftest` to `/service/test`).                                                                                                                  |
```

Comment:
```suggestion
| `decodeuricomponent`                                           | This core filter decodes URI components. For instance, it would transform `%2Fservice%2Ftest` into `/service/test`.                                                                                                                  |
```

Reply:
Thanks for the rewording, I'd just add "UTF-8 encoded string" as follows: 
`For instance, it would transform a UTF-8 encoded string`%2Fservice%2Ftest` into `/service/test`.                              `